h/feeds/render.py

```diff
@@ -1,9 +1,20 @@
+import functools
+
 from pyramid import renderers
 
+from h.features import flag_enabled
 from h.feeds import atom
 from h.feeds import rss
 
 
+def annotation_url(request, annotation):
+    """Return the HTML permalink URL for the given annotation."""
+    if flag_enabled(request, 'direct_linking'):
```

Comment:
With a view to getting this merged sooner rather than later, I'd suggest simply backing out the changes to the RSS feeds. We can always make them use "direct links" later.